### PR TITLE
Adding middle pricing tier for en-IE [DNM]

### PR DESCRIPTION
**Pull request by jamesshedden**

### CODE REVIEW

**Comment by clippy** [comment]
**I have manually checked the visuals in:**

- [ ] Chrome
- [ ] IE11
- [ ] Firefox
- [ ] Safari
- [ ] iPad

File: app/pages/pricing/pricing.en.js [hunk 1]
```diff
@@ -16,23 +16,128 @@ export default class PricingEn extends React.Component {
...
+                          Per transaction, plus <Message pointer='pricing.pro_monthly_fee' /> a month
+                        </p>
+                      </div>
+                      <Link to='contact_sales' query={{ s: 'pricing' }} className='btn btn--hollow u-size-full'>Contact sales</Link>
+                    </div>
+                  </div>
+                </div>
+              </Translation>
+
+              {/* PRICING TIERS FOR IRELAND */ }
```
Comment (jamesshedden) [inline app/pages/pricing/pricing.en.js]: @adeclosset I've separated pricing for Ireland into its own block entirely for the time being - look for this comment in the code when you're trying to find where all the Ireland pricing details are

**Comment by jamesshedden** [comment]
@adeclosset Here's the PR for changing `en-IE` pricing details - let me know if any problems/questions! :fire:

File: app/pages/pricing/pricing.en.js [hunk 2]
```diff
@@ -75,51 +180,77 @@ export default class PricingEn extends React.Component {
...
+                          <span className='u-color-accent'>&#10004;</span> Collect details by phone and paper
+                        </li>
+                      </ul>
+                      <hr />
+                      <div className='u-padding-Vs'>
+                        <h2 className='u-text-heading u-text-light u-color-dark-gray u-text-center u-text-xl'>
+                          0,20 - 0,60 €
+                        </h2>
+                        <p className='u-text-heading u-color-dark-gray u-text-center u-text-xs u-margin-Tn u-margin-Vm'>
+                          Per transaction, plus 100 € a month
```
Comment (adeclosset) [inline app/pages/pricing/pricing.en.js]: @jamesshedden - had to hardcode this one. Is that ok?

File: app/pages/pricing/pricing.en.js [hunk 3]
```diff
@@ -75,51 +180,77 @@ export default class PricingEn extends React.Component {
...
+                          <span className='u-color-accent'>&#10004;</span> Dedicated training and customer success
+                        </li>
+                      </ul>
+                      <hr />
+                      <div className='u-padding-Vs'>
+                        <h2 className='u-text-heading u-text-light u-color-dark-gray u-text-center u-text-xl'>
+                          0,01 - 0,60 €
+                        </h2>
+                        <p className='u-text-heading u-color-dark-gray u-text-center u-text-xs u-margin-Tn u-margin-Vm'>
+                          Per transaction, plus 250 € a month
```
Comment (adeclosset) [inline app/pages/pricing/pricing.en.js]: @jamesshedden - had to hardcode this one. Is that ok?

**Comment by adeclosset** [comment]
@jamesshedden - done, now checking with the team (will cc you)

**Comment by adeclosset** [comment]
@jamesshedden - all good with this, ready to ship content-wise. It does say that it's out-dated, could you help?

**Comment by jamesshedden** [comment]
@adeclosset Hey sorry been away from computer this morning - sorting this out this afternoon, will ship when it's ready :+1:

**Pull request merged**